CHANGELOG.md

```diff
@@ -6,6 +6,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+<!-- Template - Please preserve this order of sections
+## [Unreleased] - Brief description
+[Unreleased]: https://github.com/open-oni/open-oni/compare/v#.#.#...dev
```

Comment:
Might want to switch the template's compare URL here to Datura

Reply:
AHHH I have done that more than once